test/common/constants.ts

```diff
@@ -1,27 +1,27 @@
 // Test values for ARBITRUM MAINNET
-export const TEST_ORDER_ID1 = 'CLOSEDORD170141183460469231731687303715884105727-100-0';
-export const TEST_ORDER_ID2 = 'CLOSEDORD170141183460469231731687303715884105727-7000--1';
-export const TEST_ORDER_ID3 = 'CLOSEDORD170141183460469231731687303715884105728-100--1';
-export const TEST_ORDER_ID4 = 'CLOSEDORD170141183460469231731687303715884105728-200--1';
-
-export const TEST_POSITION_ID1 = 'CLOSED-POS170141183460469231731687303715884105727-100-1';
+export const TEST_ORDER_ID1 = 'CLOSEDORD13888427117130454197-100-1';
+export const TEST_ORDER_ID2 = '0x7dbfede3fb67992ceefab54d8d485278d7cd205a5d3c3de99c628651b5f88b32';
```

Comment:
why we change this constants? the 0x are not valid snx ordersIds

Reply:
with the latest  subgraph v1.013 it's the order is we are using